docs/en/kubernetes/Metrics-On-Kubernetes.md

```diff
@@ -263,20 +263,19 @@ scrape_configs:
 {% endcollapsible %}
 {% endaccordion %}
 
-#### Get Metrics Snapshot
+### Get Metrics Snapshot
 
 You can send an HTTP request to the Prometheus endpoint of an Alluxio process to get a snapshot of the metrics in Prometheus format.
 ```console
 $ kubectl exec <COMPONENT_HOSTNAME> -c <CONTAINER_NAME> -- curl 127.0.0.1:<COMPONEMT_WEB_PORT>/metrics/prometheus/
-
-# For example, get the metrics of the leading master with default web port 19999
+// For example, get the metrics of the leading master with default web port 19999
```

Comment:
`#` is the comment character for bash; `//` would break this

Reply:
![image](https://github.com/Alluxio/alluxio/assets/20586062/c1a6538e-13c0-4c9f-a5bd-333cc75be80f)
![image](https://github.com/Alluxio/alluxio/assets/20586062/c1d4eeb9-9a78-42c8-804a-a6323a3de35a)
oh i only changed it to // because on the docs code section it doesn't look like a comment (see images) is there a different comment character?

Reply:
i ended up making them each separate code blocks.. it actually looks okay